gnocchi/rest/aggregates/operations.py

```diff
@@ -150,6 +154,47 @@ def handle_binary_operator(nodes, granularity, timestamps,
     return granularity, timestamps, values, is_aggregated
 
 
+def handle_ternary_operator(nodes, granularity, timestamps,
+                            initial_values, is_aggregated, references):
+    op = nodes[0]
+    g1, t1, v1, is_a1 = evaluate(nodes[1], granularity, timestamps,
+                                 initial_values, is_aggregated, references)
+    g2, t2, v2, is_a2 = evaluate(nodes[2], granularity, timestamps,
```

Comment:
i think this will fail if you pass `None` for the min? can you add a test to verify?

Reply:
Yes, currently if we pass None to min and a number to max the operation is getting invalid.